packages/bpk-component-infinite-scroll/src/withInfiniteScroll.js

```diff
@@ -75,8 +75,8 @@ const propTypes = {
   loaderIntersectionTrigger: PropTypes.oneOf(['small', 'half', 'full']),
   onScroll: PropTypes.func,
   onScrollFinished: PropTypes.func,
-  renderLoadingComponent: PropTypes.func,
```

Comment:
Both of these are actually treated as functions, not React components so this is not correct. That said I don't think there's a down side to start treating them as components

Reply:
Ah yes I see, maybe it does make sense to keep it as that given how its used in comparios to other things